rest/model/project_event.go

```diff
@@ -97,6 +98,13 @@ func DbProjectVarsToRestModel(vars model.ProjectVars) APIProjectVars {
 	}
 }
 
+func DbProjectVarsFromRestModel(vars APIProjectVars) model.ProjectVars {
```

Comment:
do these need to be public?

Reply:
Yes because they're in the model package and used in the route package. That being said we could alternatively not use them/just copy them manually, it's not a complex function.

Reply:
shouldn't we use to ToService/FromService methods for these conversions. 

shouldn't all of the conversion stuff be hidden (potentially) in the data connector implementation?

Reply:
This was already here but you're right ToService/FromService may be more conventional.

Reply:
I think it makes sense to make this change